session.go

```diff
@@ -12,18 +12,25 @@ requests (cookies, auth, proxies).
 
 import (
 	"bytes"
-	"errors"
-	"encoding/json"
 	"encoding/base64"
+	"encoding/json"
+	"encoding/xml"
+	"errors"
+	"fmt"
 	"io/ioutil"
 	"log"
 	"net/http"
 	"net/url"
+	"path/filepath"
+	"runtime"
 	"strings"
 	"time"
 )
 
-import ()
+const (
```

Comment:
Sure. I thought these could be used as enum constants e.g.
```
    s := napping.Session{
       Encoding: napping.XML,
    }
```

By default Encoding is 0, which corresponds to the JSON enum constant.